go.mod

```diff
@@ -1,37 +1,135 @@
 module github.com/Axway/agents-mulesoft
 
-go 1.16
+go 1.17
```

Comment:
I feel like I had some issues building apigee with 1.17, not sure if I still will